cva6/sim/cva6.py

```diff
@@ -426,7 +426,7 @@ def run_assembly(asm_test, iss_yaml, isa, target, mabi, gcc_opts, iss_opts, outp
   # ISS simulation
   for iss in iss_list:
     run_cmd("mkdir -p %s/%s_sim" % (output_dir, iss))
-    log = ("%s/%s_sim/%s.log" % (output_dir, iss, asm))
+    log = ("%s/%s_sim/%s_%s.log" % (output_dir, iss, asm, test_iteration))
```

Comment:
Isn’t `test_iteration` an integer? So maybe `%d` is better there?

```suggestion
    log = ("%s/%s_sim/%s_%d.log" % (output_dir, iss, asm, test_iteration))
```

Reply:
Thanks for the review. I will updated the file accordingly.